Solutions/Azure Active Directory/Analytic Rules/AccountCreatedandDeletedinShortTimeframe.yaml

```diff
@@ -42,15 +42,15 @@ query: |
   | extend CreatedByUser = tostring(InitiatedBy.user.userPrincipalName), CreatedByIPAddress = tostring(InitiatedBy.user.ipAddress)
   | extend CreatedByApp = tostring(InitiatedBy.app.displayName)
   | project Creation_TimeGenerated, Deletion_TimeGenerated, TimeDelta, UserPrincipalName, DeletedByUser, DeletedByIPAddress, DeletedByApp, CreatedByUser, CreatedByIPAddress, CreatedByApp, Creation_AdditionalDetails = AdditionalDetails, Creation_InitiatedBy = InitiatedBy, Creation_TargetResources = TargetResources, Deletion_AdditionalDetails, Deletion_InitiatedBy, Deletion_TargetResources
-  | extend timestamp = Deletion_TimeGenerated, CustomAccountEntity = UserPrincipalName, IPCustomEntity = DeletedByIPAddress
+  | extend timestamp = Deletion_TimeGenerated
 entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
-        columnName: CustomAccountEntity
+        columnName: UserPrincipalName
```

Comment:
Fullname is not a strong entity identifier for Account (its actually a legacy mapping) where we have a UPN we should split in into Name and UPN suffix and have both as entities - https://learn.microsoft.com/en-us/azure/sentinel/entities-reference